src/main/java/de/tum/in/www1/artemis/web/rest/dto/metrics/MapEntryDTO.java

```diff
@@ -0,0 +1,10 @@
+package de.tum.in.www1.artemis.web.rest.dto.metrics;
+
+/**
+ * DTO for a map entry (key -> value).
+ *
+ * @param key   the key of the entry
+ * @param value the value of the entry
+ */
+public record MapEntryDTO(long key, long value) {
```

Comment:
This should either have a more specific name of use generics to allow different attribute types as well

Reply:
We can't use this DTO to fetch data via JPA queries if it contains generics. I agree that we should find a solution in the future.